test/lib/mayaUsd/render/vp2RenderDelegate/testVP2RenderDelegateDisplayLayers.py

```diff
@@ -147,6 +147,7 @@ def testDisplayLayers(self):
         cmds.setAttr('layer1.drawInfo.hideOnPlayback', False)
 
         # templated
+        cmds.displayRGBColor("templateActive", 1.0, 0.69, 0.69)
```

Comment:
Unclear what this additional line tests.

Reply:
I added this line to explicitly specify the color of the templated object in this test. For some reason, when I did some other changes in this file, this test started failing because the templated color was different, equal to the vaule set in a different place of this unit test, specifically from line 265: cmds.displayRGBColor("templateActive", 0.9, 0.3, 0.7)

Reply:
Oh, that mean tests are not entirely independent of each other? That could cause flakiness. OTOH, not the focus of this change...

Reply:
Yes, and this is not the first time I hit this behaviour that something that should have a default value may not necessarily have it in the end